src/planning.py

```diff
@@ -298,34 +303,39 @@ def _run_low_level_search(task: Task, option_model: _OptionModelBase,
         cur_idx += 1
         if option.initiable(state):
             try:
-                next_state, _ = option_model.get_next_state_and_num_actions(
-                    state, option)
+                next_state, num_actions = \
+                    option_model.get_next_state_and_num_actions(state, option)
             except EnvironmentFailure as e:
                 can_continue_on = False
                 # Remember only the most recent failure.
                 discovered_failures[cur_idx - 1] = _DiscoveredFailure(e, nsrt)
             else:  # an EnvironmentFailure was not raised
                 discovered_failures[cur_idx - 1] = None
+                num_actions_per_option[cur_idx - 1] = num_actions
```

Comment:
looks correct to me, but have you verified by printing out this array in an env with nontrivial options, like pybullet_blocks or cover_multistep?